### PR TITLE
NM-234: first phase of cognito implementation

### PR DESCRIPTION
**Pull request by olgazavodska**

### CODE REVIEW

File: controllers/teamdetail.github.controller.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ const fortnights = 13;
 const defaultStartDate = new Date(defaultEndDate).setDate(new Date(defaultEndDate).getDate() - fortnights * 14);
 
 function getTeamDetailWeekly(req, res, next) {
+    //const authToken = utilsService.getAuthToken(req);
```
Comment (luky89) [inline controllers/teamdetail.github.controller.js:10]: for further use?

File: services/utils.service.js [hunk 2]
```diff
@@ -66,13 +67,46 @@ function parseTimestampToDateISO(timestamp) {
...
+        if (clientId == null) {
+            const error = new Error("Authorization header is invalid. User is not authenticated.");
+            error.status = 401;
+            throw error;            
+        }
+        return clientId; // temporary using clientId as before
+    }
+}
+
+function isNumber(x) {
```
Comment (luky89) [inline services/utils.service.js:96]: do we need encapsulate isNaN function?

File: controllers/teamdetail.github.controller.js (hunk 1 above)
Comment (olgazavodska) [reply controllers/teamdetail.github.controller.js]: yes

File: services/utils.service.js (hunk 2 above)
Comment (olgazavodska) [reply services/utils.service.js]: you are right, I don't need to encapsulate it

**Pull request merged**